helpers/templates/custom_tarball/LICENSE

```diff
@@ -0,0 +1 @@
+APACHE-2.0
```

Comment:
We shouldn't default to any license since we don't have the copyright to do so. We just fetch it from GitHub, see: https://github.com/cloudposse/packages/blob/master/tasks/Makefile.package#L113